pkg/validator/properties.go

```diff
@@ -0,0 +1,18 @@
+package validator
+
+import (
+	"github.com/magiconair/properties"
+)
+
+type PropValidator struct{}
+
+// Validate implements the Validator interface by attempting to
```

Comment:
```suggestion
// PropValidator implements the Validator interface by attempting to
```

Reply:
All the other validators have the same method doc, specifying the method Validate instead of the Validator struct.

Eg: https://github.com/Boeing/config-file-validator/blob/main/pkg/validator/yaml.go#L7-L11

Shall I make the change for all or keep the current doc format.

Reply:
Nope, you're correct - I was just reading it wrong. No action required - good catch!